serialzy/serializers/base_model.py

```diff
@@ -69,13 +69,45 @@ def deserialize_from_dir(source: BinaryIO, read_from_dir) -> Any:
         return model
 
 
+def unpack_model_file(source: BinaryIO, destination: Path) -> None:
+    with destination.open("wb") as handle:
+        while True:
+            data = source.read(8096)
```

Comment:
копипаст кода сериализатора, там стоит эта чиселка

Reply:
I choose 64kb because models files are about 500mb data and default buffer size of 8kb sounds small. So says my intuition,  if you have better heuristic I'm open to suggestions :)